Procfile.dev

```diff
@@ -1,2 +1,3 @@
 web: env RUBY_DEBUG_OPEN=true bin/rails server -p ${PORT:-3000}
+postgres: docker-compose up postgres
```

Comment:
This was missing to start up with `bin/dev`

Reply:
This doesn’t seem right to me.

My mental mode has been: there are two methods of running this app, Procfile OR docker compose. Meaning, if you have docker installed and want to use docker compose, you just run “docker compose up” and everything starts: puma, postgres, and redis. But if you don’t want to install docker and you prefer to install postgres and redis through brew (or whatever means to get them installed directly on your dev machine) then you use the Procfile. So having the Procfile execute docker doesn’t seem right.

Reply:
you're right. Will revert.
I've been transitioning my local setup recently and I haven't found my ideal config yet.